lib/classifier-reborn/backends/bayes_redis_backend.rb

```diff
@@ -1,4 +1,8 @@
-require 'redis'
+begin
+  require 'redis'
+rescue LoadError
+  puts 'The redis gem is required to use the redis backend.'
```

Comment:
Yes, I think raising exception with helpful message would be a better way. It will automatically use the STDERR channel and will allow the application terminate they way it should be.